app/lib/features/settings/widgets/settings_menu.dart

```diff
@@ -24,216 +23,227 @@ class SettingsMenu extends ConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final currentRoute = ref.watch(currentRoutingLocation);
     final size = MediaQuery.of(context).size;
+
     Color? colorSelected(Routes route) => currentRoute == route.route
         ? AppTheme.brandColorScheme.secondary
         : null;
 
     TextStyle titleStylesSelected(Routes route) {
-      return TextStyle(
-        color: colorSelected(route),
-      );
+      return TextStyle(color: colorSelected(route));
     }
 
-    final shouldGoNotNamed = isDesktop || size.width > 770;
+    final shouldGoNotNamed = isDesktop && size.width > 770;
 
-    return Scaffold(
-      appBar: AppBar(
-        title: Text(
-          'Settings',
-          style: Theme.of(context).textTheme.titleLarge,
+    final isSuperInviteEnable =
+        ref.watch(hasSuperTokensAccess).valueOrNull == true;
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        _settingMenuSection(
+          context: context,
+          sectionTitle: 'Account',
+          children: [
+            _settingMenuItem(
+              context: context,
+              iconData: Atlas.key_monitor_thin,
+              iconColor: colorSelected(Routes.settingSessions),
+              title: 'Sessions',
+              subTitle: 'Your active device sessions',
```

Comment:
```suggestion
              subTitle: 'Your active devices',
```

let's simplify this.